api/base/serializers.py

```diff
@@ -490,17 +491,54 @@ def format_filter(self, obj):
 
     # Overrides HyperlinkedIdentityField
     def to_representation(self, value):
-        urls = super(RelationshipField, self).to_representation(value)
-        if not urls:
+        request = self.context.get('request', None)
```

Comment:
Why was this method changed so dramatically? I'm assuming something in `HyperlinkedIdentityField.to_representation` changed between DRF versions 3.1.1 and 3.3.3?